spec/spec_helper.rb

```diff
@@ -1,12 +1,32 @@
 require 'coveralls'
-
+require_relative 'setup_database'
 ENV['ENVIRONMENT'] = 'test'
 ENV['RACK_ENV'] = 'test'
 
 require 'capybara'
+require 'capybara/rspec'
 require 'rspec'
 require 'rake'
+require 'pg'
 
 require_relative '../makersbnb'
 
 Capybara.app = MakersBnb
+
+RSpec.configure do |config|
+
+  # config.before(:each) { Rake::Task['test_database_setup'].execute }
```

Comment:
love the use of Rake::Task

wondering if we should use this to set up the test database or if you've used a different method?

Reply:
We've commented this out for now since we haven't set up the appropriate Rakefile and associated tasks. Definitely can do once we set Rakefile up though!